File: 프로그래머스/2/12949. 행렬의 곱셈/행렬의 곱셈.py
def solution(arr1, arr2):
    answer = []
    for row in arr1:
        t =[]
        for dx in range(len(arr2[0])):
            temp = 0
            for dy in range(len(row)):
                temp+=row[dy]*arr2[dy][dx]
            t.append(temp)
        answer.append(t)
    return answer

File: 프로그래머스/2/12949. 행렬의 곱셈/행렬의 곱셈.js
function solution(arr1, arr2) {
    const r1 = arr1.length
    const c1 = arr1[0].length
    const r2 = arr2.length
    const c2 = arr2[0].length
    const answer = [...new Array(r1)].map((_,i)=>new Array(c2).fill(0))
    
    for (let i=0;i<r1;i++){
        for(let j=0;j<c2;j++){
            for(let k=0;k<r2;k++){
                answer[i][j]+=(arr1[i][k]* arr2[k][j])
            }
        }
    }

    return answer;
}